spring-cloud-vault-core/pom.xml

```diff
@@ -22,6 +22,11 @@
 			<artifactId>spring-cloud-context</artifactId>
 		</dependency>
 
+        <dependency>
```

Comment:
Thanks @spencergibb.  I'll adjust on the next update.  Is there configuration beyond the code formatter mentioned in the docs I can apply to better comply with your standards?